lib/govuk_navigation_helpers/related_navigation_sidebar.rb

```diff
@@ -70,7 +70,15 @@ def related_statistical_data_sets
     end
 
     def related_topics
-      build_links_for_sidebar(@content_item.related_topics)
+      topics = build_links_for_sidebar(@content_item.related_topics)
+      if related_mainstream_topic
```

Comment:
You could condense this down to

```ruby
topics << related_mainstream_topic if related_mainstream_topic
topics << related_mainstream_parent_topic if related_mainstream_parent_topic
```

Reply:
or even do it in one line:

```ruby
(topics <<  related_mainstream_topic << related_mainstream_parent_topic).compact
```

Reply:
Great! Changes made [here](https://github.com/alphagov/govuk_navigation_helpers/pull/106/files#diff-d681d9743587da4bdc011c5858f7e450R73)